### PR TITLE
Show "Double click" when mouse hovers on project name

### PR DESCRIPTION
**Pull request by MyunghyunNero**
## Description

If user hover the mouse over the 'project name' in the 'project list', show tooltip text "Double click"
projectName mouse hoverd, show Double Click

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by soimkim** [comment]
@MyunghyunNero , Would you correct the indentation?

**Comment by soimkim** [comment]
Dear @MyunghyunNero , let's merge commits into one!

**Comment by MyunghyunNero** [comment]
thank you for telling me!
merged into one commit.😂

**Comment by MyunghyunNero** [comment]
I changed the way to inject the function from outside.
Can you please review it?

File: src/main/webapp/WEB-INF/views/admin/project/list-js.jsp [hunk 1]
```diff
@@ -957,7 +957,9 @@
 					alertify.error('<spring:message code="msg.common.valid2" />', 0);
 				}
 			});
-		}
+		},toolTipDoubleclick: function () {
+            return 'title="' + "Double click" + '"';
+        }
```
Comment (FRESH-TUNA) [inline src/main/webapp/WEB-INF/views/admin/project/list-js.jsp:962]: Indentation of 961~962 line should be fixed

Comment (MyunghyunNero) [reply src/main/webapp/WEB-INF/views/admin/project/list-js.jsp]: thank you for code review!! 😊👍
i'll fix it!!

**Pull request merged**